15_perms_lab/permslab.txt

```diff
@@ -2,14 +2,40 @@
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
 
-  - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
-
-  - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
-
-  - change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
-
-2. create a new directory named "test" and inside an empty file named "secret". Change permissions of the file "secret" to ---------. Can you delete the file secret? can you delete the folder test? why?
+ 
+
+chmod a-wx src
+
+ - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
+
+  
+
+562 include
+
+- change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
+
+  -
+
+rchmodf a-x go-rwx Doc

```

Comment:
That would change permissions on Doc folder itself, which is not what the question asked. We need to change the permissions of the files inside Doc, but not of the folder.

The rationale is a directory MUST have execute permission. Without it you can't do anything in that directory.

So correct answer here is `chmod 600 Doc/*`